pkg/systemd/logs.jsx

```diff
@@ -57,6 +57,7 @@ const timeFilterOptions = [
     { key: "boot", value: 0, toString: () => _("Current boot"), },
     { key: "boot", value: "-1", toString: () => _("Previous boot") },
     { key: "since", value: "-24hours", toString: () => _("Last 24 hours") },
+    { key: "since", value: "-3days", toString: () => _("Last 3 days"), default: true },
```

Comment:
Was there some particular reasoning about 3 days? ISTM that keeping it simple and showing the last day is a bit more intuitive -- Usually, when you look at logs, you want to investigate something recent. But bikeshedding, of course

Reply:
I was under the impression it was decided here https://github.com/cockpit-project/cockpit/issues/15085#issuecomment-891630590 I think the reasoning for selecting 3 days is that one day doesn't give enough information.

Reply:
After some more thought I've changed it to the last 24 hours.